app/helpers/template_helpers.py

```diff
@@ -63,21 +141,24 @@ def render_template(template, **kwargs):
     google_tag_manager_context = get_google_tag_manager_context()
     cdn_url = f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
     base_url = get_census_base_url(theme, language_code)
-    contact_us_url = get_contact_us_url(language_code, base_url)
-    footer_urls = get_footer_urls(language_code, base_url, theme)
     include_csrf_token = request.url_rule and "POST" in request.url_rule.methods
     account_service_url = cookie_session.get(
         "account_service_url", f"{CENSUS_BASE_URL}en/start"
     )
+    sign_out_url = url_for("session.get_sign_out")
+    content_urls = get_static_content_urls(language_code, base_url, theme)
+    footer_context = get_footer_context(
+        language_code, theme, content_urls, sign_out_url
+    )
```

Comment:
The theme might not exist at this point so it needs a default of "census". At the moment, without a cookie and non census themes do not render a footer.